code/src/components/AddTodoForm.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { todoTasks } from 'reducers/todoTasks'
+
+// defined our state from react
+export const AddTodoForm = () => {
+    const [todoText, setTodoText] = useState('')
+    const dispatch = useDispatch()
+    //When the input changes it will trigger the event which is passed on and the name/input will change.
+
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        dispatch(todoTasks.actions.addTodo(todoText))
+        setTodoText('')
+    }
+
+    return (
+        <div className="inputField">
+            <form className=""
+                onSubmit={handleSubmit}>
```

Comment:
Keep an eye on your indentation. Things start going a little wild from here down. Also, don't forget to change your editor to 2-spaces for indentation instead of 4!